src/chart/parameter/ParameterSelectionChart.tsx

```diff
@@ -65,6 +66,7 @@ export const NeoParameterSelectionChart = (props: ChartProps) => {
         settings={props.settings}
         allParameters={allParameters}
         compatibilityMode={compatibilityMode}
+        multiSelector={multiSelector}
       />
     );
   } else if (type == 'Relationship Property') {
```

Comment:
relationship property breaks when selecting a number:
E.g. ()-[:REVIEWED{rating:5}]->().
Seems to be the case for both with/without multiselect